build_projects/dotnet-host-build/TestTargets.cs

```diff
@@ -49,7 +49,9 @@ public static BuildTargetResult RestoreTests(BuildTargetContext c)
             var dotnet = DotNetCli.Stage0;
             CleanBinObj(c, Path.Combine(Dirs.RepoRoot, "test"));
 
-            dotnet.Restore()
+            dotnet.Restore(
+                    "--fallbacksource", Dirs.CorehostLocalPackages,
```

Comment:
This should be unnecessary as the test projects themselves are not intended to target the latest generated host.